src/views/registration-js/attendees/content.php

```diff
@@ -68,5 +68,15 @@
 				?>
 			<?php endwhile; ?>
 		</script>
+		<?php
+			foreach ( $fields as $field ) {
+				/** @see \Tribe__Tickets_Plus__Meta__Field__Abstract_Field::get_escaped_inline_javascript */
+				$field_javascript = $field->get_escaped_inline_javascript();
+
+				if ( ! empty( $field_javascript ) ) {
+					echo '<script>' . $field_javascript . '</script>';
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$field_javascript'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$field_javascript'.

<sub>hash: 7371b06bedae56f4a09a77dfe3b2a17e</sub>